build/openresty/patches/ngx_lua-0.10.25_01-dyn_upstream_keepalive.patch

```diff
@@ -677,6 +692,14 @@ index af4da73..407c115 100644
 +}
 +
 +
++static void
++ngx_http_lua_balancer_notify_peer(ngx_peer_connection_t *pc, void *data,
```

Comment:
From the peer notify callback use case of Nginx https://github.com/nginx/nginx/blob/f255815f5d161fab0dd310fe826d4f7572e141f2/src/stream/ngx_stream_upstream_round_robin.c#L689, I suggest following Nginx style to define new macro `NGX_HTTP_LUA_BALANCER_NOTIFY_EMPTY_RETRY` to differentiate many other notify type in future.

Reply:
apply